crates/host/src/wasmbus/mod.rs

```diff
@@ -810,7 +811,7 @@ impl Host {
             labels: RwLock::new(labels),
             ctl_nats,
             rpc_nats: Arc::new(rpc_nats),
-            host_config: config,
+            host_config: config.clone(),
```

Comment:
optional nit: if you assign `config.max_execution_time` to a variable above this `let host = Host {}` block, you won't need to clone here